Bottom: 4a3ca1cd2a1573b3f75a3a1350e7a4ccf7c80a0e
Top:    8cd7f911e13e19ccaffe4a921971e76795cc8353
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 17:26:06 +0000

Disable small edge EPSILON hack for computing edge normals


---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index ee8938b..5781a61 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -231,7 +231,7 @@ object3d_to_step_body_fragment (step_file *step,
           dir_y = dv->y - ov->y;
           dir_z = dv->z - ov->z;
 
-#if 1
+#if 0
           /* XXX: This avoids the test file step_outline_test.pcb failing to display properly in freecad when coordinates are slightly rounded */
           if (dir_x < EPSILON && -dir_x < EPSILON &&
               dir_y < EPSILON && -dir_y < EPSILON &&
